cmd/skopeo/utils.go

```diff
@@ -28,6 +28,9 @@ func contextFromGlobalOptions(c *cli.Context, flagPrefix string) (*types.SystemC
 			return nil, err
 		}
 	}
+	if c.BoolT(flagPrefix + "no-creds") {
```

Comment:
One more thing, please make this fail if both `"creds"` and `"no-creds"` are specified.